components/storage/blockdevice/COMPONENT_SPIF/TESTS/block_device/spif/main.cpp

```diff
@@ -49,8 +49,9 @@ void basic_erase_program_read_test(SPIFBlockDevice &block_device, bd_size_t bloc
     _mutex->lock();
 
     // Make sure block address per each test is unique
-    static unsigned block_seed = 1;
-    srand(block_seed++);
+    static unsigned block_seed = SPIF_TEST_NUM_OF_THREADS;
+    srand(block_seed);
+    block_seed += 2;
 
```

Comment:
Please see #9855
SPIF and QSPIF tests were removed (merged fully into general block device tests)

Reply:
Do we need to pick 9855 as well?

Reply:
Sounds good to me